Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/AddBasket-AddNonprofitsViewController.m

```diff
@@ -47,11 +50,20 @@ - (void)getAllNonprofitsFromParse {
 - (IBAction)createButtonTapped:(id)sender {
     NSLog(@"%@", self.basket.nonprofits);
     [self.basket saveInBackgroundWithBlock:^(BOOL succeeded, NSError * _Nullable error) {
-       //save basket to parse
-        //clear all data, have navigation controller reset to Create Basket
-        //switch tab controller to index 0
+        if (succeeded) {
+            AddBasketViewController *rootVC = self.navigationController.viewControllers.firstObject;
+            rootVC.basketNameTextField.text = @"";
+            rootVC.basketCategoryTextField.text = @"";
+            rootVC.basketDescriptionTextView.text = @"Describe your basket in 100 words or less.";
```

Comment:
All these properties should be set internally by `AddBasketViewController`. You can define a `resetForm` method or something in the `AddBasketViewController` class and call it here if needed.

Reply:
Got it! This is a good point